yarl/_url.py

```diff
@@ -63,7 +63,6 @@ class CacheInfo(TypedDict):
 
 
 class _SplitResultDict(TypedDict, total=False):
```

Comment:
Let's avoid unrelated formatting changes.
```suggestion
class _SplitResultDict(TypedDict, total=False):

```